app/script/entity/Conversation.js

```diff
@@ -167,30 +167,42 @@ z.entity.Conversation = class Conversation {
     this.hasActiveCall = ko.pureComputed(() => (this.hasLocalCall() ? this.call().isActiveState() : false));
     this.hasJoinableCall = ko.pureComputed(() => (this.hasLocalCall() ? this.call().canJoinState() : false));
 
-    this.unreadEvents = ko.pureComputed(() => {
+    this.unreadEventsCount = ko.observable(0);
+    this.unreadMessagesCount = ko.observable(0);
+    this.hasUnreadSelfMention = ko.observable(false);
+
+    this.unreadEvents = ko.computed(() => {
```

Comment:
Let's not create a `computed` here. 
As a rule of thumb, you can keep in mind that `subscribe` or `computed` should not live in an entity.
They should live in the upper layers of the app as much as possible (viewmodel for example).

The problem with subscribing in an entity is that we don't have the mechanism to tear down the subscription, and it will live on until the browser is reloaded.

I would suggest that unreadEvents is a `pureComputed` and that `unreadEventsCount`, `unreadMessageCount` and `hasUnreadSelfMention` are also `pureComputed` that depends on `unreadEvents`.